crossdock/client/tchclient/thrift.go

```diff
@@ -21,11 +21,16 @@
 package tchclient
 
 import (
+	"reflect"
+	"strings"
 	"time"
 
 	"github.com/yarpc/yarpc-go/crossdock-go"
+	"github.com/yarpc/yarpc-go/crossdock/client/gauntlet"
+	"github.com/yarpc/yarpc-go/crossdock/client/pointer"
 	"github.com/yarpc/yarpc-go/crossdock/client/random"
 	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/echo"
+	"github.com/yarpc/yarpc-go/crossdock/thrift/gen-go/gauntlet_apache"
```

Comment:
What governs whether you choose to use underscore, hyphen, or no delimiter at all?